README.md

```diff
@@ -10,10 +10,53 @@ For more information, see the [website][1].
 Our [Protecting infrastructure secrets with Keywhiz][2] blog post is worth
 reading, as it provides some useful context.
 
-## Contributing
+## Develop
 
 See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches.
 
+Build keywhiz: 
+
+    # Build keywhiz for H2
+    mvn install -P h2
+
+    # Build keywhiz for MySQL
+    mvn install -P mysql
+
+Run Keywhiz:
+
+    java -jar server/target/keywhiz-server-*-shaded.jar [COMMAND] [OPTIONS] 
+
+Useful commands to get started are `migrate`, `db-seed` and `server`. Use with
+`--help` for a list of all available commands. Use with `[COMMAND] --help` to
+get help on a particular command.
+
+For example, to run Keywhiz with an H2 database in development mode:
+
+    export SERVER_JAR=server/target/keywhiz-server-*-shaded.jar
+    export KEYWHIZ_CONFIG=server/target/classes/keywhiz-development.yaml.h2
+
+    # Initialize dev database (H2)
```

Comment:
I believe the development config has a static master key and certificate, so this will just work(TM).